src/footer/footergrid.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+
+import { useStyles } from 'footer/styles'
+
+export const Footergrid = () => <Grid container alignContent={'center'} justify={'center'} item xs={12} sm={4}>
```

Comment:
The `xs`, `sm`, etc. props are only valid on `<Grid item/>` components, so these are useless here. Let's remove them.

Reply:
Done!